src/content/content.js

```diff
@@ -58,7 +58,10 @@ async function findApplicables(location, openInSidebar = true) {
 }
 
 async function main() {
-	setupSidebar()
+	const support = await browser.storage.local.get("sidebarActionSupported");
+	if (support.sidebarActionSupported === false) {
+		setupSidebar();
+	}
```

Comment:
um, I actually use the in-page version in FF, so I'll revert it..

Reply:
Can we make this an extension setting? It causes some issue in wikibase #62 that i couldn’t get behind

Reply:
yeah, we need to do settings.  
but also does #62 not happen for you in Chrome, but only in FF with internal sidebar enabled??

Reply:
The bug didn’t occur in chrome